packages/falcon-sensor/overlay/app-readme.md

```diff
@@ -0,0 +1,7 @@
+# CrowdStrike Falcon
+
+The [CrowdStrike Falcon](https://www.crowdstrike.com/) platform is purpose-built to stop breaches
+via a unified set of cloud-delivered technologies that prevent all types of attacks — including
+malware and much more.
+
+For more information, please visit [https://www.crowdstrike.com](https://www.crowdstrike.com)
```

Comment:
```suggestion
CrowdStrike Container Security comes complete with vulnerability management, continuous
threat detection and response, and runtime protection, combined with compliance
enforcement and automated continuous integration/continuous delivery (CI/CD) pipeline security, enabling
DevOps teams to stay secure while building in the cloud.

For more information, please visit [https://www.crowdstrike.com/cloud-security-products/falcon-cloud-workload-protection/](https://www.crowdstrike.com/cloud-security-products/falcon-cloud-workload-protection/)
```

Reply:
@shawndwells can you please open a ticket and address this upstream?

Reply:
This has been fixed upstream